debian/rules

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/make -f
+%:
+	dh $@ --with python3 --buildsystem=pybuild
+
+override_dh_auto_clean:
+	dh_auto_clean
+	rm -f debian/key-mapper.service
+
+override_dh_auto_install:
+	dh_auto_install
+	# remove the systemd-service in /usr/lib/systemd and replace it with dh-systemd
+	rm -rf debian/key-mapper/usr/lib/systemd/
+	cp data/key-mapper.service debian/key-mapper.service
+	# move udev-rule from /etc to /lib for packaging
+	mkdir -p debian/key-mapper/lib
+	mv debian/key-mapper/etc/udev debian/key-mapper/lib/udev
```

Comment:
You can avoid all this by adding symlinks permanently in `debian`:

```
key-mapper.service -> ../data/key-mapper.service
key-mapper.udev -> ../data/key-mapper.rules
```

This ensures that the maintainer scripts get the added snippets required to correctly handle services.

I also added a patch in `debian/patches` to remove the two lines in `setup.py` which install the files.

Reply:
Sounds good. @sezanzeb wants to change the udev-path in setup.py anyway. I'd write a patch for the service when this gets migrated to quilt.

Reply:
Ah, didn't realize that udev also has a debhelper. nice

Reply:
Yes, `dh_installudev`.